src/gui/components/TextInput.zig

```diff
@@ -393,6 +412,16 @@ pub fn inputCharacter(self: *TextInput, character: u21) void {
 	}
 }
 
+pub fn inputString(self: *TextInput, utf8EncodedString: []const u8) void {
```

Comment:
The functionality you actually need is `setString`, why not implement that function instead?

Reply:
setString would also clear it, making the explicit clearing at the callsite unnecessary.